app/tumblr.hoon

```diff
@@ -167,34 +167,34 @@
 ::  also produce a result so that the request doesn't hang?
 ::
 ++  sigh-tang
-  |=  {way/wire tan/tang}
-  ^-  {(list move) _+>.$}
+  |=  [way=wire tan=tang]
+  ^-  [(list move) _+>.$]
   %-  (slog >%gh-sigh-tang< tan)
   [[ost.hid %diff null+~]~ +>.$]
 ::
 ++  poke-tumblr-get-posts
-  |=  {iden/@t idst/@u}
+  |=  [iden=@t idst=@u]
   ^-  (quip move +>.$)
-  =/  pax/path  ?:(=(iden %dashboard) /x/user/dashboard /x/blog/[iden]/posts/(scot %u idst))
-  ?>  ?=({care *} pax)
+  =/  pax=path  ?:(=(iden %dashboard) /x/user/dashboard /x/blog/[iden]/posts/(scot %u idst))
+  ?>  ?=([care *] pax)
   :_  +>.$  :_  ~
   (read:connector ost.hid (places (welp /poke/read pax)) i.pax t.pax)
 :: 
 ::  We can't actually give the response to pretty much anything
 ::  without blocking, so we just block unconditionally.
 ::
 ++  peer
-  |=  pax/path
+  |=  pax=path
   :_  +>.$
   %+  turn  
     (prey pax hid)
-  |=  {o/bone *}
+  |=  [o=bone *]
   :^    o
       %diff
     %json
   (joba connect+(jape "success"))
 ++  peek
-  |=  tyl/path
+  |=  tyl=path
   ^-  (unit (unit (pair mark *)))
-  ~ ::``noun/[ren tyl]
+  ~ ::``noun=[ren tyl]
```

Comment:
@vvisigoth/[tumblr-urbit-api](https://github.com/vvisigoth/tumblr-urbit-api)

@mattlevan It's really confusing that development for all of these repos happens in different places. This should at least be documented -- actually, maybe they already are in the app readmes.